deps/Makefile

```diff
@@ -116,3 +117,9 @@ jemalloc: .make-prerequisites
 	cd jemalloc && $(MAKE) lib/libjemalloc.a
 
 .PHONY: jemalloc
+
+fast_float_c_interface: .make-prerequisites
+	@printf '%b %b\n' $(MAKECOLOR)MAKE$(ENDCOLOR) $(BINCOLOR)$@$(ENDCOLOR)
+	cd fast_float_c_interface && $(MAKE) CFLAGS="$(OPT) $(STD) $(WARN) $(DEBUG)" LDFLAGS="$(LDFLAGS)"
```

Comment:
Here, we pass the variable CFLAGS to the next Makefile. We don't pass the individual OTP, STD, WARN and DEBUG variables.

Reply:
yeah I forgot to update this. good catch. fixing it.